klant/src/main/kotlin/com/ritense/klant/domain/KlantSearchFilter.kt

```diff
@@ -18,14 +18,12 @@ package com.ritense.klant.domain
 
 data class KlantSearchFilter(
     val bsn: String? = null,
-    val kvk: String? = null,
```

Comment:
Deze zou dus moeten blijven bestaan

Reply:
Er ging iets niet lekker met het mergen al. Denk dat hier het slachtoffer ervan te zien is. Ik ga het even goed zetten